### PR TITLE
remove unnecessary fields in vercel.json from docs

### PR DESCRIPTION
**Pull request by styfle**

### CODE REVIEW

File: docs/vercel.json [hunk 1]
```diff
@@ -1,10 +1,5 @@
...
   "$schema": "https://openapi.vercel.sh/vercel.json",
-  "build": {
-    "env": {
-      "ENABLE_EXPERIMENTAL_COREPACK": "1",
-      "ENABLE_ROOT_PATH_BUILD_CACHE": "1",
-      "FORCE_RUNTIME_TAG": "canary"
-    }
-  }
+  "installCommand": null,
+  "buildCommand": null
```
[the remark this answers is not in the document]
Comment (styfle) [reply docs/vercel.json:4]: This is the equivalent of turning of the "Override" section here
image

[the remark this answers is not in the document]
Comment (mehulkar) [reply docs/vercel.json:4]: ok, will deal with those separately. Maybe we can turn off the overrides entirely.

**Review by mehulkar — APPROVED** [review]
🚢 it

**Comment by mehulkar** [comment]
`Test / Turbopack Rust testing on macos ` and `Test / Check next-swc ` failing. These are unrelated. These probably shouldn't be running here anyway, probably because of global package.json change). 

Force merging

**Pull request merged**